src/components/AllInfoOfParticularPost/AllInfoOfParticularPost.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Button,TextArea,Modal,Input} from 'semantic-ui-react';
+const allinfoofparticularpost=(props)=>{
+    return(
+            <>
+                <Modal trigger={<Button basic color='blue'>More details...</Button>}>
+                    <Modal.Header>Blog</Modal.Header>
+                    <Modal.Content>
+                    <Modal.Description>
+                        <label>TITLE:</label>
+                        {/* <Input name="title" onChange={(event)=>props.onEditInput(props.id,event)} value={props.title}/><br></br><br></br> */}
```

Comment:
remove unwanted code or commented code.

Reply:
Sir,I have removed the unwanted commented code.